src/datasources/notifications/entities/notification-devices.entity.ts

```diff
@@ -0,0 +1,48 @@
+import { NotificationSubscription } from '@/datasources/notifications/entities/notification-subscription.entity';
+import type { UUID } from 'crypto';
+import {
+  Check,
+  Column,
+  Entity,
+  Unique,
+  OneToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('push_notification_devices')
+@Unique('device_uuid', ['device_uuid'])
+@Check('device_type', 'device_type IN ("ANDROID", "IOS", "WEB")')
```

Comment:
Can we set this to the values of the enum via TypeORM?

Reply:
Do you mean we should dynamically create the string? TypeOrm `Check` only accepts string. I wouldn't dynamically create the string as that kills the readability here.

Reply:
It would be nice to somehow have them linked. We aren't otherwise certain that our enum and this are the same.

Reply:
How about changing the field type in the database to ENUM? We can handle that in a separate PR.
P.S. I didn't modify the database fields, as it was beyond the scope of this PR.